Makefile

```diff
@@ -61,7 +61,7 @@ $(BUILD_DIR)/configure: $(BUILD_DIR)/config.m4
 	$(Q) (cd $(BUILD_DIR); phpize && sed -i 's/\/FAILED/\/\\bFAILED/' $(BUILD_DIR)/run-tests.php) # Fix PHP 5.4 exit code bug when running selected tests (FAILED vs XFAILED)
 
 $(BUILD_DIR)/Makefile: $(BUILD_DIR)/configure
-	$(Q) (cd $(BUILD_DIR); ./configure)
+	$(Q) (cd $(BUILD_DIR); ./configure --enable-datadog-profiling=$(PHP_DATADOG_PROFILING))
```

Comment:
Just putting a note here for others: this is strict about truthy values so this has to be `PHP_DATADOG_PROFILING=yes`.

Reply:
If anyone is aware of an existing m4 macro that handles some set of truthy values like `yes`, `true`, and `on`, please let me know.